### ISSUE TITLE
Right clicking kicks the client

### ISSUE THREAD

**Issue by notfoundname** [KryptonMC/Krypton#74]
### Description
When player right clicks any block (or tries to place some), he gets kicked with an error in console.

### Steps to reproduce
1. Start the server;
2. Connect to it;
3. Right click on a block / place a block
4. :)

### Expected behaviour
Client should not get kicked

### Details
https://paste.gg/p/anonymous/c22beb302ef34da6904283b838617838

* Server version: Krypton 0.60.4
* Java version: 
  openjdk 17.0.1 2021-10-19
  OpenJDK Runtime Environment Temurin-17.0.1+12 (build 17.0.1+12)
  OpenJDK 64-Bit Server VM Temurin-17.0.1+12 (build 17.0.1+12, mixed mode, sharing)

**Comment by BomBardyGamer** [comment]
Well this is a strange one. I think block placing is still unfinished, but I will investigate this and see if I can find the problem. Thanks for the report!

**Comment by StarL0st** [comment]
Quite late but I think it has to do with the block handling, possibly having to do with some sort of illegal interaction for notchian clients that causes them to be kicked, from what I can see, it has to do that it is receiving an illegal block pos?, I dont know much kotlin but I'm willing to help

**Comment by BomBardyGamer** [comment]
Yeah that sounds about right. As I said, block placing is still unfinished, and still needs to be done. It hasn't been done yet because there is a lot for us to do when blocks get placed, a lot of actions to process, and I need to figure out what actually needs to be done and when, and I've been focusing on other things.
